### PR TITLE
Modify rule S3949: Remove CBDE from C#, add VB.NET

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Fixes https://github.com/SonarSource/sonar-dotnet/issues/7139

### CODE REVIEW

File: rules/S3949/csharp/rule.adoc [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+----
+public long Transform(int value)
+{
+    if (value <= 0)
+    {
+        return value;
+    }
+    long number = int.MaxValue;
+    return number + value;  // Noncompliant
+}
```
Comment (Tim-Pohlmann) [inline rules/S3949/csharp/rule.adoc:30]:
```suggestion
public long Transform(int value)
{
    if (value <= 0)
    {
        return value;
    }
    long number = int.MaxValue;
    return number + value;
}
```

**Review by Tim-Pohlmann — CHANGES_REQUESTED** [review]

File: rules/S3949/csharp/rule.adoc (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply rules/S3949/csharp/rule.adoc]: Oops, and it was missing on the VB version

**Review by Tim-Pohlmann — APPROVED** [review]
LGTM

**Comment by pavel-mikula-sonarsource** [comment]
There are 2 new commits to review

**Review by Tim-Pohlmann — APPROVED** [review]
LGTM

**Pull request merged**